uktrade/data-hub-frontend
# Update organisation types

## Description of change
Organisation types were previously hardcoded and need to be set from a subset of business types.

## Test instructions
1. Browse to `~/companies/create`
2. Select `UK`
3. Click `Next`
4. Search for `ralph`
5. Click `I cannot find the company I am looking for`
6. Click `I still cannot find the company`, you can see the `Organisation type` radio buttons

## Screenshots
### Before
![Screenshot 2019-09-13 at 15 05 15](https://user-images.githubusercontent.com/1150417/64879315-9c065f80-d64d-11e9-8bf1-4f620979f467.png)

### After 
![Screenshot 2019-09-13 at 15 04 27](https://user-images.githubusercontent.com/1150417/64879320-9f99e680-d64d-11e9-85f9-d6fb6526a24e.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
